### DISCUSSION TITLE
Continuous Reboot

### DISCUSSION THREAD

**Discussion by xefil** [rand256/valetudo#498] [Q&A]
Hi all,
I'm having issues getting the robot rebooting different times during cleanup and cannot understand why.
I was connected via SSH and get:

```
root@rockrobo:/var/log# 
Broadcast message from root@rockrobo
        (unknown) at 15:40 ...

The system is going down for reboot NOW!
```

So something is pushing a reboot via software. How to identify the cause?

Here some infos:
```
MODEL...........: rockrobo.vacuum.v1
PRODUCTION DATE.: UNKNOWN
FIRMWARE........: 3.5.8_004028
BUILD NUMBER....: 2021010900REL
MIIO VERSION....: 3.3.9

Valetudo RE 0.10.8_armhf.deb
```

Please let me know, so is the vacuum not usable, still reboots often.

Thanks,

Simon

**Comment by rand256** [comment]
The only way valetudo RE calls a reboot is when someone performs a PUT request at `http://device_addr/api/reboot_device` URL.
If you're sure it's not the case, I guess another option could only be the manufacturer's firmware gone mad. Maybe it thinks something's going wrong in it and has no other idea than doing a restart to see if it helps. In that case I could only suggest trying a factory reset.

**Reply by xefil** [reply]
Hello!
Is there a way to check logs and identify who is performing this operation?
Would a reset loose my ssh access and I need to perform the whole replace of the firmware from the beginning?
I cannot remember if on my Gen1 I was able to access ssh without open the device. Many years ago :-)
Thanks!
Simon

**Comment by smartm0use** [comment]
Hi, sorry to revive this old thread. Have you noticed if it always happens at the same time, no matter the cleaning duration?
